benchmark-framework/src/main/java/io/openmessaging/benchmark/Benchmark.java

```diff
@@ -64,6 +64,9 @@
                 "--workers-file" }, description = "Path to a YAML file containing the list of workers addresses")
         public File workersFile;
 
+        @Parameter(names = { "-x", "--extra" }, description = "Allocate extra consumer workers")
+        boolean extraConsumers;
```

Comment:
What are the `extraConsumers` used for? It should also be included in the description

Reply:
This is for Pulsar JMS testing